spec/factories/comment.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryBot.define do
+  factory :comment do
+    commenter { Faker::Name.name }
+    body { Faker::Lorem.sentence }
+    article { build(:article) }
```

Comment:
I think here we need to use `create` instead of `build`, just in case we actually need `article` to be saved in the database